jdbc/src/java/org/apache/hive/jdbc/HiveDatabaseMetaData.java

```diff
@@ -855,19 +860,19 @@ public boolean nullPlusNonNullIsNull() throws SQLException {
   }
 
   public boolean nullsAreSortedAtEnd() throws SQLException {
-    throw new SQLFeatureNotSupportedException("Method not supported");
+    return false;
   }
 
   public boolean nullsAreSortedAtStart() throws SQLException {
-    throw new SQLFeatureNotSupportedException("Method not supported");
+    return false;
   }
 
   public boolean nullsAreSortedHigh() throws SQLException {
```

Comment:
An unit test would be useful for nullsAreSortedHigh and nullsAreSortedLow

Reply:
Added